source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -1,142 +1,105 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { Binding, ExpressionParser, EvaluationContext } from "./expression-parser";
-import * as Shared from "./shared";
+import * as CEL from "adaptive-expressions";
```

Comment:
>CEL [](start = 12, length = 3)

Nit: Should probably rename this alias/variable at this point since CEL is deprecated naming for adaptiveExpressions